README.md

```diff
@@ -355,7 +355,8 @@ render status: :forbidden
     Unless they have some meaning in the business domain, don't put methods in
     your model that just format your data (like code generating HTML). These
     methods are most likely going to be called from the view layer only, so their
-    place is in helpers. Keep your models for business logic and data-persistence
+    place is in helpers, if simple, or a seperate class if more complex.
```

Comment:
- Typo on separate
- Consider illustrating "separate class" with related pattern names to assist readers on how to proceed and what to look up next

> or a separate class (e.g. presenter, value object) if more complex

Reply:
Spelling fixed in 663f1f28922be10daf3cbd82bf9205bb6386a6ed.

I've made the guidelines more specific in bf43762.